dev_guide/deployments/advanced_deployment_strategies.adoc

```diff
@@ -164,7 +164,7 @@ In the editor, add the line `ab-example: "true"` underneath `spec.selector` and
 . Trigger a re-deployment of the first shard to pick up the new labels:
 +
 ----
-$ oc deploy ab-example-a --latest
+$ oc rollout latest ab-example-a
```

Comment:
It looks a bit strange and I'm not sure whether it works, because here is we're using `ab-example-a` without `dc/` prefix. Let me know if I need to fix it.